### PR TITLE
Design system storybook apps card

### PR DESCRIPTION
**Pull request by bhushankhope**

### CODE REVIEW

**Comment by LibbyUX** [comment]
@bhushankhope ping me on here when this is ready for another review, thanks so much for working on this!

**Comment by bhushankhope** [comment]
> @bhushankhope ping me on here when this is ready for another review, thanks so much for working on this!

@LibbyUX This is ready for review

**Review by LibbyUX — CHANGES_REQUESTED** [review]
We are super close, thank you @bhushankhope! Is it possible to remove the focus states border/stroke on active/on press? Ideally, we just see the active state on press. We should only see the outline on tab, if possible. If this isn't possible, I will remove the focus state for this button.

15B18381-614F-450A-9C9D-E91B3310C386

**Comment by LibbyUX** [comment]
@bhushankhope & I just caught up - This is not possible, so I removed the focus state in [Design System Repository](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=4806-18710&t=jhyHA9dVObHEUK0p-4) and [Design System Components](https://www.figma.com/design/BCEJn9KCIbBJ5MzqnojKQp/Design-System-Components?node-id=78-355).

**Comment by bhushankhope** [comment]
@LibbyUX I have added a dropdown to select each card. Please review. Thanks
[link](https://github.com/hubmapconsortium/hra-ui/pull/637#issuecomment-2284553142)

**Comment by bhushankhope** [comment]
> We are super close, thank you @bhushankhope! Is it possible to remove the focus states border/stroke on active/on press? Ideally, we just see the active state on press. We should only see the outline on tab, if possible. If this isn't possible, I will remove the focus state for this button.
> 
> 15B18381-614F-450A-9C9D-E91B3310C386 [ 15B18381-614F-450A-9C9D-E91B3310C386 ](https://private-user-images.githubusercontent.com/97060856/357168837-caa98cf8-1d75-41ed-a553-b9605045722f.GIF?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1NjMzMzksIm5iZiI6MTcyMzU2MzAzOSwicGF0aCI6Ii85NzA2MDg1Ni8zNTcxNjg4MzctY2FhOThjZjgtMWQ3NS00MWVkLWE1NTMtYjk2MDUwNDU3MjJmLkdJRj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEzVDE1MzAzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM1OWRmMzQxMDY2N2VlOGRkYTc3NGQyN2M1Y2Q1MTg0Mjc1Yzk4NjUwN2FkOThiZWY0YTg2NGJjYzY4ZTM3ZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gbMlFIlV0U3w_FVRFZ27SJHFwh_UB-YQkP9EsqNWii4) [ ](https://private-user-images.githubusercontent.com/97060856/357168837-caa98cf8-1d75-41ed-a553-b9605045722f.GIF?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1NjMzMzksIm5iZiI6MTcyMzU2MzAzOSwicGF0aCI6Ii85NzA2MDg1Ni8zNTcxNjg4MzctY2FhOThjZjgtMWQ3NS00MWVkLWE1NTMtYjk2MDUwNDU3MjJmLkdJRj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEzVDE1MzAzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM1OWRmMzQxMDY2N2VlOGRkYTc3NGQyN2M1Y2Q1MTg0Mjc1Yzk4NjUwN2FkOThiZWY0YTg2NGJjYzY4ZTM3ZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gbMlFIlV0U3w_FVRFZ27SJHFwh_UB-YQkP9EsqNWii4)

removed the focus state from code

**Review by LibbyUX — APPROVED** [review]
Perfect, thank you so much @bhushankhope !

File: libs/design-system/apps-card/src/lib/apps-card.component.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    .title {
+      font: var(--sys-label-large);
+      color: var(--sys-secondary);
+    }
+
+    .description {
+      grid-area: description;
+      font: var(--sys-label-medium);
+      color: #4b4b5e;
```
Comment (axdanbol) [inline libs/design-system/apps-card/src/lib/apps-card.component.scss:38]: Should come from a --sys-* variable

File: libs/design-system/apps-card/src/lib/apps-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  selector: 'hra-apps-card',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './apps-card.component.html',
+  styleUrl: './apps-card.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppsCardComponent {
+  /** URL for the icon */
+  icon = input.required<string>();
```
Comment (axdanbol) [inline libs/design-system/apps-card/src/lib/apps-card.component.ts:15]: Mark inputs as readonly

**Review by axdanbol — CHANGES_REQUESTED** [review]
Tweaks

**Review by axdanbol — APPROVED** [review]

**Pull request merged**